### PR TITLE
745 chunk size issue

### PR DESCRIPTION
**Pull request by norisalsaadie**

### CODE REVIEW

File: src/domain/file/services/file-service.ts [hunk 1]
```diff
@@ -199,45 +216,59 @@ export class FileService {
...
-                fileUploadManager,
-                bytesRead,
-                uploadStatusFunction,
-            );
-            parts.push({ ETag, PartNumber: partNumber });
+            currentBytes = currentBytes + chunk.length;
+            chunksToSend.push(chunk);
             chunks.push(chunk);
-            partNumber++;
+            if (currentBytes >= s3MinChunkSize || bytesRead === file.size) {
```
Comment (diversemix) [inline src/domain/file/services/file-service.ts:233]: Do we check / handle the fact that file.size might not be the same as the total chunk sizes? ... Even if its just for logging purposes - after the for loop exits, check that we've sent the expected size?

Comment (norisalsaadie) [reply src/domain/file/services/file-service.ts:233]: This should be handled in the higher level middleware for apollo.

File: src/domain/file/services/file-service.ts [hunk 2]
```diff
@@ -70,17 +74,28 @@ export class FileService {
...
         try {
             const params = await this.s3.uploadPart(partParams).promise();
-            await statusFunction(userId, file.filename, file.id, Math.floor((bytesRead / file.size) * 100));
+            await pubsub.publish('UPLOAD_STATUS', {
+                fileUploadProgress: {
+                    userId,
+                    filename: file.filename,
+                    fileId: file.id,
+                    percentage: Math.floor((bytesRead / file.size) * 100),
+                    type: FileType.MANUSCRIPT_SOURCE,
```
Comment (diversemix) [inline src/domain/file/services/file-service.ts:83]: Shouldn't this be the type passed in?

Comment (norisalsaadie) [reply src/domain/file/services/file-service.ts:83]: Yes, good point. Didn't refactor it as nicely as I had thought

Comment (norisalsaadie) [reply src/domain/file/services/file-service.ts:83]: Will push up as part of the fix for failing tests

**Comment by diversemix** [comment]
Would be good to see if we can get a test in to check for it using the correct chunk size too?

**Pull request merged**